_partials/_devops-rest-api-get-started.md

```diff
@@ -69,70 +69,62 @@ proper authentication headers.
+1. **Create a $SERVICE_SHORT using the POST endpoint**
    ```bash
-   cat > service-config.json << 'EOF'
+   curl -X POST "${API_BASE_URL}/projects/${TIGERDATA_PROJECT_ID}/services" \
+     -u "${TIGERDATA_ACCESS_KEY}:${TIGERDATA_SECRET_KEY}" \
+     -H "Content-Type: application/json" \
+     -d '{
+         "name": "my-first-service",
+         "service_type": "TIMESCALEDB",
+         "region_code": "us-east-1",
+         "replica_count": 1,
+         "cpu_millis": 1000,
+         "memory_gbs": 4
+     }'
+   ```
+   $CLOUD_LONG creates a Development environment for you. That is, no delete protection, high-availability, spooling or
+   read-replication. You see something like:
+   ```terminaloutput
    {
-     "name": "my-first-service",
-     "service_type": "TIMESCALEDB",
-     "region_code": "us-east-1",
-     "replica_count": 1,
-     "cpu_millis": 1000,
-     "memory_gbs": 4
-   }
-   EOF
+     "service_id":"asdfasdfasdf","project_id":"asdasdfasf","name":"my-first-service",
+     "region_code":"us-east-1", "service_type":"TIMESCALEDB",
+     "created":"2025-09-09T09:24:31.997767396Z", "paused":false,"status":"READY",
+     "resources":[{"id":"101240",
+     "spec":{"cpu_millis":1000,"memory_gbs":4,"volume_type":""}}],
+     "metadata":{"environment":"PROD"},
+     "endpoint":{"host":"oh.yeah.tsdb.cloud.timescale.com","port":123435},
+     "initial_password":"very-secret",
+     "ha_replicas":{"sync_replica_count":0,"replica_count":1}
+   } 
    ```
 
-1. **Submit service creation request**
-
-    1. Create the service using the POST endpoint:
-       ```bash
-       curl -X POST "${API_BASE_URL}/projects/${TIGERDATA_PROJECT_ID}/services" \
-         -H "Authorization: Basic $(echo -n "${TIGERDATA_ACCESS_KEY}:${TIGERDATA_SECRET_KEY}" | base64)" \
-         -H "Content-Type: application/json" \
-         -d @service-config.json
-       ```
-       $CLOUD_LONG creates a production environment for you. You see something like:
-       ```terminaloutput
-       {
-         "service_id":"asdfasdfasdf","project_id":"asdasdfasf","name":"my-first-service",
-         "region_code":"us-east-1", "service_type":"TIMESCALEDB",
-         "created":"2025-09-09T09:24:31.997767396Z", "paused":false,"status":"READY",
-         "resources":[{"id":"101240",
-         "spec":{"cpu_millis":1000,"memory_gbs":4,"volume_type":""}}],
-         "metadata":{"environment":"PROD"},
-         "endpoint":{"host":"oh.yeah.tsdb.cloud.timescale.com","port":123435},
-         "initial_password":"very-secret",
-         "ha_replicas":{"sync_replica_count":0,"replica_count":1}
-       } 
-       ```
-
-
-2. Save `service_id` from the response to a variable:
+1. Save `service_id` from the response to a variable:
    ```bash
    # Extract service_id from the JSON response
    export SERVICE_ID="service_id-from-response"
    ```
 
-1. **Change the environment from production to development**
+1. **Check the configuration for the **
```

Comment:
This sentence seems to trail off. Maybe it should be something like this?

```suggestion
1. **Check the configuration for the $SERVICE_SHORT**
```

Reply:
Nice spot, done.